app/views/simulator/edit.html.erb

```diff
@@ -152,12 +155,11 @@
    <!--   <div>
         <button type="button" class="quick-btn-redo logixButton" title="Redo" onclick="">
         </button> -->
-        <div class="zoom-slider">
-          <button class="zoom-slider-decrement" id="decrement">-</button>
-          <input type="range" class="form-range" min="0" max="45" step="1" id="customRange1">
-          <span id='slider_value'></span>
-          <button class="zoom-slider-increment" id="increment">+</button>
-        </div>
+        <div class="zoom-slider d-flex align-items-center">
+  <button class="zoom-slider-decrement btn btn-outline-secondary" id="decrement" aria-label="Zoom Out">-</button>
+  <input type="range" class="form-range mx-2" min="0" max="45" step="1" id="customRange1" aria-label="Zoom Level">
+  <button class="zoom-slider-increment btn btn-outline-secondary" id="increment" aria-label="Zoom In">+</button>
+</div>
```

Comment:
This is unrelated, create separate Pr for this

Reply:
Ok noted